aws/cloudformation-templates/template.yaml

```diff
@@ -265,6 +265,11 @@ Conditions:
     - !Ref IncludeSegmentDependencies
     - 'Yes'
 
+  DeployLocationStack: !Equals
+    - !Ref DeployLocationServices
```

Comment:
Please add a suitable default value for `DeployLocationServices` to the group workshop template here:

https://github.com/aws-samples/retail-demo-store/blob/master/aws/cloudformation-templates/event-engine/base-workshop.yaml